apps/Gruntfile.js

```diff
@@ -208,7 +208,8 @@ config.lodash = {
 config.sass = {
   all: {
     options: {
-      outputStyle: (MINIFY ? 'compressed' : 'nested')
+      outputStyle: (MINIFY ? 'compressed' : 'nested'),
+      includePaths: ['../shared/css/']
```

Comment:
Here's the new include path.  @wjordan is there a new version of this I need to modify as well?